### DISCUSSION TITLE
Reimplement mania `ComboPosition` configuration lookup

### DISCUSSION THREAD

**Discussion by GSculerlor** [ppy/osu#27460] [Q&A] [answered]
opening a discussion since I don't know the current priority of lazer and the planned path of mania skinning in general. 

current mania skin on lazer doesn't implement the legacy mania combo counter placed in the center of the playfield and can be configured by changing `ComboPosition`. is this expected to be as is or does `ComboPosition` lookup still need to be implemented in the future?

I have a modified debug client lying around for a personal mania flashlight training client that "implements" `ComboPosition` lookup for mania legacy skin since I need to see my combo during gameplay. the position and the animation might be slightly different from the actual result in stable since I don't have any stable code reference, but if it is expected to be implemented in the future I can clean my changes and try to make a PR. 

what I currently have on my modified debug client:

https://github.com/ppy/osu/assets/6506864/2b05aae4-6aca-400d-9226-35f373095bcf

**Comment by bdach** [comment, marked as answer]
The mania-specific legacy combo counter should be implemented as the element in the "playfield" HUD layer and the legacy lookup should be migrated to a correct position in the new skinning system. 

https://github.com/ppy/osu/pull/26249 etc. is a prerequisite.

**Reply by GSculerlor** [reply]
ah, nice to see it already being worked on. I assume this will also be applied to the mania legacy health bar as well in the follow-up changes?

**Reply by bdach** [reply]
Correct
